test/python/simple_script.py

```diff
@@ -0,0 +1,39 @@
+# ==============================================================================
+#  Copyright 2019 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ==============================================================================
+"""nGraph TensorFlow bridge TF to NG conversion logs test
+
```

Comment:
This is not a pytest. It is used by the pytest `test/python/test_tf_to_ng_conversion_logs.py` to check if appropriate `NGRAPH_TF_LOG_PLACEMENT=1` logs are being generated or not